rep-2000.rst

```diff
@@ -1179,6 +1179,120 @@ Package manager use for dependencies:
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Qt             |    5.15.13     |    5.12.10     |  5.15.9  |    5.15.16    |      5.15.8      | 5.14.1 / 5.12.5***  |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+|                                 |                                   **Linux only**                                   |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| PCL            |     1.14.0     |      N/A       |  1.12.0  |    1.14.1     |      1.13.0      |       1.10.0        |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+|                                                **RMW DDS Middleware**                                                |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Cyclone DDS    |                                               0.10.5                                                |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Fast-DDS       |                                               2.14.4                                                |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Connext DDS    |                          6.0.1.25                          |                  N/A                   |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Gurum DDS      |              4.2.0              |                                N/A                                |
++----------------+----------------+----------------+----------+---------------+------------------+---------------------+
+
+" * " means that this is not the upstream version (available on the official Operating System repositories) but a package distributed by OSRF or the community (package built and distributed on custom repositories).
+
+" ** " means that the dependency may see multiple version changes, because the dependency uses a package manager that continually updates the dependency without a stable API.
+
+" \*** " webOS OSE provides this different version.
+
+This document only captures the version at the first release of a ROS distribution and will not be updated as the dependencies move forward.
+These versions are thus a low watermark.
+
+
+Package manager use for dependencies:
+
+- Ubuntu, Debian: apt, pip
+- Windows: Chocolatey, pip
+- macOS: Homebrew, pip
+- RHEL: dnf
+- OpenEmbedded: opkg
+
+
 Build System Support:
 
 - ament_cmake
```

Comment:
We should decide the scope of "build system" here. I'm not sure if `ament_python` or `ament_cargo` qualify. I'm not even sure if `ament_cmake` really qualifies anymore.

```suggestion
- ament_cmake
- cargo
```

Reply:
I honestly have no idea.  This particular part has never made much sense to me.  I'm OK with dropping it, if you think that is better.